ql/src/java/org/apache/hadoop/hive/ql/io/orc/VectorizedOrcAcidRowBatchReader.java

```diff
@@ -959,6 +989,20 @@ public boolean next(NullWritable key, VectorizedRowBatch value) throws IOExcepti
       int ix = rbCtx.findVirtualColumnNum(VirtualColumn.ROWID);
```

Comment:
we could probably do the same optimization here

Reply:
see my previous comment for `VirtualColumn.ROWISDELETED`